sim/hunter/kill_shot.go

```diff
@@ -0,0 +1,79 @@
+package hunter
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func (hunter *Hunter) registerKillShotSpell() {
+	if hunter.HasPrimeGlyph(proto.HunterPrimeGlyph_GlyphOfKillShot) {
+		icd := core.Cooldown{
+			Timer:    hunter.NewTimer(),
+			Duration: time.Second * 6,
+		}
+		ksReset := hunter.RegisterAura(core.Aura{
+			Label: "Kill Shot Glyph",
+			Icd: &icd,
+			Duration: time.Second,
+			OnGain: func(aura *core.Aura, sim *core.Simulation) {
+				hunter.KillShot.CD.Reset()
+			},
+		})
```

Comment:
What is the reason this is implemented as a 1 second aura instead of ust calling `hunter.KillShot.CD.Reset()` instead of applying an aura that does it